cirq/circuits/circuit_operation_test.py

```diff
@@ -167,31 +196,70 @@ def test_with_params():
         _ = cirq.resolve_parameters(op_base, cirq.ParamResolver(param_dict))
 
 
-def test_repetition():
-    a, b = cirq.LineQubit.range(2)
-    # This circuit has a modulus of 8.
-    circuit = cirq.FrozenCircuit(cirq.H(a), cirq.CX(a, b))
-    op_base = cirq.CircuitOperation(circuit)
-    assert op_base.repeat(1) is op_base
+def cartesian_product_of_string_lists(list1: List[str], list2: List[str]):
+    return [f'{first}-{second}' for first in list1 for second in list2]
```

Comment:
Are you not able to import this from the file directly? i.e.
`from cirq.circuits.circuit_operation import cartesian_product_of_string_lists`

If not, this should be updated to match that one.

Reply:
Done. Thanks for catching this!